packages/datatrak-web/src/features/Leaderboard/LeaderboardTable.tsx

```diff
@@ -115,23 +99,23 @@ export const LeaderboardTable = ({ userRewards, user, leaderboard }: Leaderboard
             <HeaderCell>Score</HeaderCell>
           </HeaderRow>
         </TableHead>
-        <Body $rowCount={leaderboard?.length}>
+        <TableBody>
           {leaderboard?.map(({ userId, firstName, lastName, coconuts }, i) => {
             const isActiveUser = user && user.id === userId;
             return (
-              <Row key={userId}>
+              <TableRow key={userId}>
                 <Cell>{i + 1}</Cell>
                 <Cell $isActiveUser={isActiveUser}>
                   {firstName} {lastName}
                 </Cell>
-                <Cell $isActiveUser={isActiveUser}>{coconuts}</Cell>
-              </Row>
+                <Cell $isActiveUser={isActiveUser}>{coconuts.toLocaleString()}</Cell>
```

Comment:
`isActiveUser` is only used to set a heavier font weight on entire table rows at a time. Personally, I’d argue its use could be promoted up one level to the row, rather than using it on the cell. I gave it a crack, but am not quite comfortable enough with how to use transient props yet, so this PR doesn’t include this suggestion.

Reply:
It's definitely possible but I think styling it either way is totally fine in this case

Reply:
Hardly a major issue, so let’s keep as-is